cellrank/tl/kernels/_pseudotime_kernel.py

```diff
@@ -120,6 +120,12 @@ def compute_transition_matrix(
         :class:`cellrank.tl.kernels.PseudotimeKernel`
             Makes :attr:`transition_matrix` available.
         """
+        if frac_to_keep < 0 or frac_to_keep > 1:
```

Comment:
Remove this - it's not necessary (will be caught later in the PT scheme).

Reply:
I added it in both places to receive the error as soon as possible and in case someone calls the `HardThresholdScheme` independently. But either way is fine by me. I removed it.